pkgs/development/python-modules/clang/common.nix

```diff
@@ -14,12 +13,13 @@ buildPythonPackage rec {
     inherit pname sha256 version;
   };
 
-  checkInputs = [ nose ];
+  # The pypi distribution does not have any tests.
```

Comment:
use `fetchFromGitHub` then

Reply:
Also the comment is not required. It is very standard to do this, so much standard that there are no questions asked in review if you fetch from GitHub/etc instead of pypi.

Reply:
There is no convenient GitHub repo for this package to fetch from unfortunately.

Reply:
But the thing we're using right now "is a non-official packaging directly from the debian packages for the purpose of pypi package." We should really just build Python bindings at the same time as clang.

Reply:
Yeah, there's been some discussion on this (https://github.com/NixOS/nixpkgs/issues/89214#issuecomment-922336699). But for the moment that discussion seems to have stalled, so I propose that we move forward with sub-optimal pypi bindings and then switch to nice nix-built ones once they're ready.